vue-begin/src/components/Employees.vue

```diff
@@ -0,0 +1,187 @@
+<template>
+  <div>
+    <h1> {{ msg }}</h1>
+
+    <h2>Add new employee</h2>
+    <v-flex class="text-xs-center py-3">
+      <v-btn @click="showDialog(true,employee)" color="primary">
+        <v-icon>add</v-icon> Add Employee
+      </v-btn>
+    </v-flex>
+
+    <h2>Display all employees</h2>
+    <v-list three-line>
+      <v-list-tile v-for="employee in paginatedEmployees"
+        class="employees"
+        v-bind:key="employee.id"
+      >
+        <v-list-tile-content>
+          <v-list-tile-title>
+            Name: {{ employee.employee_name }}
+          </v-list-tile-title>
+          <v-list-tile-title>
+            Salary: {{ employee.employee_salary | currency }}
+          </v-list-tile-title>
+          <v-list-tile-title>
+            Age: {{ employee.employee_age }}
+          </v-list-tile-title>
+        </v-list-tile-content>
+        <v-list-tile-action>
+        <v-btn icon @click="showDialog(true, employee)">
+          <v-icon>edit</v-icon>
+        </v-btn>
+      </v-list-tile-action>
+      <v-list-tile-action>
+        <v-btn icon @click="remove(employee.id)">
+          <v-icon>delete</v-icon>
+        </v-btn>
+      </v-list-tile-action>
+      </v-list-tile>
+    </v-list>
+    <div class="text-xs-left">
+      <v-pagination
+        class="pagination-list"
+        v-model="currentPage"
+        :length="Math.floor(employees.length/employeesPerPage)"
```

Comment:
`length` should be declared as a `computed` property

Reply:
```
 :length= length()
 następnie
      length() {
        return Math.floor(employees.length/employeesPerPage);
      }
```

Reply:
`:length= length`
`
      length() {
        return Math.floor(this.employees.length/this.employeesPerPage);
      }
`

Reply:
ok